sdk/src/main/java/com/uid2/UID2Manager.kt

```diff
@@ -155,6 +155,22 @@ public class UID2Manager internal constructor(
         }
     }
 
+    /**
+     * Generates a new identity with the given (hashed) email address.
+     *
+     * Once set, assuming it's valid, it will be monitored so that we automatically refresh the token(s) when required.
+     * This will also be persisted locally, so that when the application re-launches, we reload this Identity.
+     */
+    public fun generateIdentity(emailHash: String): Unit = afterInitialized {
```

Comment:
just wondering, is it worth renaming to setIdentityFromEmail or similar to stay inline with the JS SDK convention?
https://unifiedid.com/docs/guides/publisher-client-side#configure-the-uid2-sdk-for-javascript